ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -521,10 +548,11 @@ def write_summary_file(run_id, out_dir, analysis_result_per_file):
         total_match_cnt += file_match_cnt
         total_expect_cnt += file_expect_cnt
 
-    out_file.write("-----------------------------\n")
-    out_file.write("TOTAL MATCHES:    %d\n" % total_match_cnt)
-    out_file.write("TOTAL EXPECTED MATCHES:    %d\n" % total_expect_cnt)
-    out_file.write("-----------------------------\n")
+    out_file.write("-----------------------------------\n")
+    out_file.write("TOTAL MATCHES:                  %d\n" % total_match_cnt)
+    out_file.write("TOTAL EXPECTED MATCHES:         %d\n" % total_expect_cnt)
+    out_file.write("TOTAL EXPECTED MISSING MATCHES: %d\n" % len(unused_regex_messages))
```

Comment:
>TOTAL EXPECTED MISSING MATCHES:  [](start = 20, length = 32)

Do you need to change back the grep command which modified by Maggie, in this PR?

Reply:
Yeah, could your please change the loganalyzer_end.yml which process the different result for validation?

Reply:
Sure, done